edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -734,6 +734,29 @@ export default {
       const serviceCIDR = this.serverConfig['service-cidr'] || '';
 
       return clusterCIDR.includes(':') || serviceCIDR.includes(':');
+    },
+
+    appsOSWarning() {
+      if (this.mode !== _EDIT ) {
+        return null;
+      }
```

Comment:
Why do we only want this to happen on edit? Couldn't the user specify two pools with different OSs when creating?

Reply:
They could do that while creating the cluster, but there wouldn't be any installed apps to worry about breaking in that case